hawc/apps/common/templates/common/dynamic_form.html

```diff
@@ -1,2 +1,2 @@
-
+{% load crispy_forms_tags %}
```

Comment:
need to figure out which `dynamic_form.html` to remove? In hawc there is
`templates/common/dynamic_form.html` and `templates/common/widgets/dynamic_form.html` which are the same

Reply:
done! sorry I had a comment I sent to you but realized too late it was "pending" and never sent. I deleted the widgets one.

* fe71fbee8 remove duplicative widget and template